<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/Attic/CQExpressionMmlWidget.ui,v $ </comment>
<comment>   $Revision: 1.6 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: pwilly $ </comment>
<comment>   $Date: 2009/09/23 12:22:23 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQExpressionMmlWidget</class>
 <widget class="QWidget" name="CQExpressionMmlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Expression-Mml Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="mpWidgetStackExpressionMml">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="mpExpressionPage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="CQExpressionWidget" name="mpExpressionWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_ExpPage">
         <item>
          <widget class="QToolButton" name="mpBtnExpressionObject">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>select object</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mpBtnViewExpression">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>view expression</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="mpSpacerExpressionObject">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mpMmlPage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CQMmlScrollView" name="mpMmlScrollView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_MmlPage">
         <item>
          <widget class="QToolButton" name="mpBtnEditExpression">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>edit expression</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="mpBtnSaveExpression">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolTip">
            <string>save expression</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="mpSpacerMmlObject">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CQExpressionWidget</class>
   <extends>QTextEdit</extends>
   <header>CQExpressionWidget.h</header>
   <slots>
    <signal>valid(bool)</signal>
    <slot>slotSelectObject()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>CQMmlScrollView</class>
   <extends>QFrame</extends>
   <header>CQMmlScrollView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnExpressionObject</sender>
   <signal>clicked()</signal>
   <receiver>mpExpressionWidget</receiver>
   <slot>slotSelectObject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnViewExpression</sender>
   <signal>clicked()</signal>
   <receiver>CQExpressionMmlWidget</receiver>
   <slot>updateWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpExpressionWidget</sender>
   <signal>valid(bool)</signal>
   <receiver>mpBtnViewExpression</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnEditExpression</sender>
   <signal>clicked()</signal>
   <receiver>CQExpressionMmlWidget</receiver>
   <slot>slotGoExpressionWidget()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnSaveExpression</sender>
   <signal>clicked()</signal>
   <receiver>CQExpressionMmlWidget</receiver>
   <slot>slotSaveExpression()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateWidget()</slot>
  <slot>slotGoExpressionWidget()</slot>
  <slot>slotSaveExpression()</slot>
 </slots>
</ui>
